nodewatcher/modules/frontend/ip_space/static/css/ipspace.css

```diff
@@ -0,0 +1,42 @@
+.noselect {
+    -webkit-touch-callout: none; /* iOS Safari */
+    -webkit-user-select: none; /* Safari */
+    -khtml-user-select: none; /* Konqueror HTML */
+    -moz-user-select: none; /* Firefox */
+    -ms-user-select: none; /* Internet Explorer/Edge */
+    user-select: none; /* Non-prefixed version, currently
+                      supported by Chrome and Opera */
+}
+.silky-smooth{
+    border-color: rgb(12,70,156);
+    border-width: 1px;
+    border-style: solid;
+    width: 100%;
+    transform-origin: top left;
+}
+/*Author of tipsy: https://github.com/jaz303 */
```

Comment:
What is tipsy and where it did come from? Is there an upstream repository?

Reply:
Can this just be included, instead of  copied?

Reply:
I included repository information, and moved tipsy css to own file, however this is not exact copy of the tipsy repository as it is modified to work correctly with the d3.js.